lib/6to5/transformation/transform.js

```diff
@@ -68,6 +68,7 @@ _.each({
   computedPropertyNames:     require("./transformers/es6-computed-property-names"),
   destructuring:             require("./transformers/es6-destructuring"),
   defaultParameters:         require("./transformers/es6-default-parameters"),
+  forOfFast:                 require("./transformers/optional-for-of-fast"),
```

Comment:
Transformer position matters, the other optional ones are at the bottom because so they catch all prior transforms.